Gemfile

```diff
@@ -7,3 +7,6 @@ gemspec
 source 'https://oss:Px2ENN7S91OmWaD5G7MIQJi1dmtmYrEh@gem.mutant.dev' do
   gem 'mutant-license'
 end
+
+gem 'mutant',       git: 'https://github.com/mbj/mutant', branch: 'add/ruby-3.0'
+gem 'mutant-rspec', git: 'https://github.com/mbj/mutant', branch: 'add/ruby-3.0'
```

Comment:
Chicken & egg is active here, will switch over to a released mutant once there is an unparser release ;)